code-challenges.test.js

```diff
@@ -25,27 +25,69 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+//         expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")      
+//     })
+// })
+//ReferenceError: divisableByThree is not defined
+
+// const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+
+// // b) Create the function that makes the test pass.
+
+// const divisibleByThree = (object) => {
+//     //1) create a function divisible by three and call the objects with their key values.
+//     const number = object.number // made a new variable "number" to call the objects and their key value.
+//         if(number % 3 === 0){ // write a conditional statement to determine if that object can be divisible by 3 using a modulo.
+//             return `${number} is divisible by three`
+//         } else  {
+//             return `${number} is not divisible by three`
+//         }
+// }
+// console.log(divisibleByThree(object1))//output : 15 is divisble by 3
+// console.log(divisibleByThree(object2))// output: 0 is divisble by 3
+// console.log(divisibleByThree(object3))// output: -7 is not divisble by 3
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
-// b) Create the function that makes the test pass.
 
 // Pseudo code:
+//input: a function and objects
+//output: determine if object is divisible by 3
+//(wrote the psuedo code along with the actual code)
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
+   describe ("capitalizeWords", () => {
+    it("takes the element of the array and modifies the first character", () => {
+        expect(capitalizeWords(randomNouns1)).toEqual["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+        expect(capitalizeWords(randomNouns2)).toEqual["Temperature", "Database", "Chopsticks", "Mango"]
+    })
+   })
+   //ReferenceError: toUpper is not defined
+
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
-
-// Pseudo code:
\ No newline at end of file
+const capitalizeWords = (strings) => { // takes an array as input
+    return strings.map((value) => { // goes through each word in the array, capitalizes the first letter and adds the capitalized word to the function
+        return value[0].toUpperCase() + value.slice(1)
```

Comment:
The pseudo code here is confusing. What do you mean when you say the capitalized first letter is added to the function? The function is `capitalizeWords` and this logic lives in the function. You can reword this like so:
- capitalizes first letter of the word, then adds the rest of the word to it